public/home.html

```diff
@@ -16,5 +16,10 @@ <h2>H o m e P a g e</h2>
     <p>
         <a href="home2.html" target="_self">next page</a>
         </p>
+        <form action="/somewhere_else" method="POST">…</form>
```

Comment:
The home page should not have a form. It should have three links - one to create a quiz, one to delete a quiz, one to take a quiz.

Reply:
ok, so i need to now make the "delete a quiz" and change create.html to /somewhere else

Reply:
The handout describes the create and take a quiz; the delete a quiz page works similarly. It asks "enter the quiz name you want to delete" with a form with an input control, and submit goes to a URL we don't have yet, and we might as well have it POST to `/remove`.

Focus on Create and Take paths, then Delete last.

Reply:
> ok, so i need to now make the "delete a quiz" and change create.html to /somewhere else

That's backwards. `somewhere_else` is a PLACEHOLDER that you change.

Reply:
idk what a place holder is XD :,( can you relate it to something I've implemented? or dumb down the definition? 
also, I'm looking at the hand out from week 10, I see what you're saying, but I'm not sure I know how to do it. Are some of the things i need to do on the beginners HTML tutorial?

Reply:
https://www.dictionary.com/browse/placeholder